File: PRAK301-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
num1 = int(input("Masukkan angka pertama: "))
num2 = int(input("Masukkan angka kedua: "))

if num1 > num2:
    num1, num2 = num2, num1

print("Angka terurut:", num1, num2)

File: PRAK301-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>

int main() {
    int num1, num2, temp;

    printf("Angka : ");
    scanf("%d %d", &num1, &num2);

    if (num1 > num2) {
        
        temp = num1;
        num1 = num2;
        num2 = temp;
    }

    printf("Angka terurut: %d %d\n", num1, num2);

    return 0;
}
